docs/en/aws-deploy-elastic-serverless-forwarder.asciidoc

```diff
@@ -333,7 +335,8 @@ NOTE: To deploy the forwarder directly without using SAR, refer to <<aws-serverl
 [role="screenshot"]
 image::images/aws-serverless-forwarder-create-function.png[Create Elastic Serverless Forwarder Lambda function within SAR]
 +
-. Complete the **Application settings** according to <<aws-serverless-forwarder-define-deploy-parameters>>.
+. Complete the **Application settings** according to <<aws-serverless-forwarder-define-deploy-parameters>>. 
+Please specify the parameters even if they already exist in the `config.yaml` file.
```

Comment:
```suggestion
You must specify the parameters even if they already exist in the `config.yaml` file.
```
should be also mentioned explicitly that there must be defined at least one of the parameters: `ElasticServerlessForwarderSQSEvents`/`ElasticServerlessForwarderS3SQSEvents`/`ElasticServerlessForwarderKinesisEvents`/ `ElasticServerlessForwarderCloudWatchLogsEvents`, depending on the used input type ?